src/server/actions/Organizations.ts

```diff
@@ -1,6 +1,32 @@
 import dbConnect from '@/utils/db-connect';
-import OrganizationSchema from '@/server/models/Organization';
+import OrganizationSchema from '../models/Organization';
+import { Organization } from '@/types/dataModel/organization';
+import { CreateOrganizationRequest } from '@/types/dataModel/organization';
 import { OrganizationEntity } from '@/types/dataModel/organization';
+/**
+ * Create an organization
+ * @param CreateOrganizationRequest requires the name of the organization
+ * @returns _id The Id of the organization
+ */
+
+export async function createOrganization(
+  request: CreateOrganizationRequest
+): Promise<string> {
+  try {
+    const connection = await dbConnect();
+    const organization = await OrganizationSchema.create(request);
+
+    connection.connection.on('error', (err) => {
+      throw new Error(err.code);
```

Comment:
Not sure if this code does anything. When I test it, the code never hits here and a full error object is still thrown

Reply:
I tested it just now, and it doesn't do anything.